### ISSUE TITLE
Stunt rally 64bit Raspberry Pi OS

### ISSUE THREAD

**Issue by 333337lee** [Botspot/pi-apps#511]
Stunt rally Shows up in the menu of PiApps.

/home/pi/pi-apps/apps/Stunt Rally/install-64: line 28: cmake command not found 
Failed to run ‘cmake ..’!

thanks for all the great work.

Lee

**Comment by Itai-Nelken** [comment]
it seems that the dependencies aren't being installed, that means that my code here isn't working:
https://github.com/Botspot/pi-apps/blob/a7da08ce38b7b532b4209d5b2a4b87f8060375b5/apps/Stunt%20Rally/install-64#L11-L13
will try to find out why, maybe using `pkg-install` and `purge-installed` is a better idea.
***
@Botspot can `pkg-install` and `purge-installed` be used on the same install script (to install the build dependencies and then uninstall them when done)?
***
@333337lee can you run this on rpios 64bit and post the output please?
```bash
DEPENDS="cmake git build-essential libboost-wave-dev libboost-system-dev libboost-filesystem-dev libboost-thread-dev libogre-1.9-dev libmygui-dev libsdl2-dev libogg-dev libvorbis-dev libenet-dev libopenal-dev libopenal-dev libbullet-dev libbullet-extras-dev" 
TOINSTALL="$(dpkg -l $DEPENDS 2>&1 | awk '{if (/^D|^\||^\+/) {next} else if(/^dpkg-query:/) { print $6} else if(!/^[hi]i/) {print $2}}' | tr '\n' ' ')" 
echo $TOINSTALL
```

**Comment by Botspot** [comment]
> @Botspot can `pkg-install` and `purge-installed` be used on the same install script (to install the build dependencies and then uninstall them when done)?

They've never been used like that before, but I don't see why not.

**Comment by Itai-Nelken** [comment]
[Stunt Rally.zip](https://github.com/Botspot/pi-apps/files/6212843/Stunt.Rally.zip)
this should work.
at the end I realized that the `install-32` script uses `pkg-install` but the `uninstall` script doesn't use `purge-installed` so I added `purge-installed` to the `install-32` script and replaced my method with `pkg-install` in the `install-64` script.

ao this zip fixes the dependencies not being uninstalled, and 64bit not working.
